pkg/scheduler/apis/config/v1alpha1/types.go

```diff
@@ -55,11 +55,22 @@ type SchedulerConfiguration struct {
 	// settings for the proxy server to use when communicating with the apiserver.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection,omitempty"`
 	// LeaderElection defines the configuration of leader election client.
-	LeaderElection LeaderElectionConfiguration `json:"leaderElection,omitempty"`
+	LeaderElection componentbaseconfigv1alpha1.LeaderElectionConfiguration `json:"leaderElection,omitempty"`
 	// LogLevel is the level/severity for the logs. Must be one of [info,debug,error].
 	LogLevel string `json:"logLevel,omitempty"`
-	// Server defines the configuration of the HTTP server.
+	// LogFormat is the output format for the logs. Must be one of [text,json].
+	LogFormat string `json:"logFormat,omitempty"`
```

Comment:
Should this be an optional field (pointer, `omitempty`), that is properly defaulted like in the gardenlet config API?

Reply:
As discussed, for the sake of consistency I left it the way it is. Defaulting works fine if `logFormat` is omitted.